### PR TITLE
Paste without formatting (2)

### PR DESCRIPTION
**Pull request by Soreine**
This is a better implementation of #30 
This simply expose a new plugin `slate-raw-paste`, which handles `shift+mod+v`.
The logic is cleaner, because it's packaged in a single plugin.

### CODE REVIEW

File: packages/slate-react/src/plugins/rawPaste.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        // During onPaste, we don't have access to the pressed keys,
+        // so we detect it here
+        isRawPaste = Hotkeys.isRawPaste(event);
+    }
+
+    /*
+     * On paste.
+     */
+
+    function onPaste(event: Event, change: Change, editor: EditorContainer) {
```
Comment (Banb4n) [inline packages/slate-react/src/plugins/rawPaste.tsx]: Same here, `editor` is not used

File: packages/slate-react/src/plugins/rawPaste.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+ */
+function RawPastePlugin(): Plugin {
+    // True if the current keyown event happened following a special paste
+    let isRawPaste = false;
+
+    /*
+     * On key down.
+     */
+
+    function onKeyDown(event: Event, change: Change, editor: EditorContainer) {
```
Comment (Banb4n) [inline packages/slate-react/src/plugins/rawPaste.tsx]: Why do you go 'change` & `editor` as they are not used?

File: packages/slate-react/src/index.ts [hunk 3]
```diff
@@ -11,7 +9,7 @@ import getEventTransfer from './utils/get-event-transfer';
 import setEventTransfer from './utils/set-event-transfer';
 
 export * from './interfaces';
-export * from './plugins';
+export * from './plugins/index';
```
Comment (Banb4n) [inline packages/slate-react/src/index.ts]: Why do you have added `index` ?

File: packages/slate-react/src/plugins/rawPaste.tsx (hunk 2 above)
Comment (Soreine) [reply packages/slate-react/src/plugins/rawPaste.tsx]: It's the complete signature of the plugin methods. That's what they are passed.
Having them stated everywhere avoids confusion (consistency) and can be convenient.

File: packages/slate-react/src/index.ts (hunk 3 above)
Comment (Soreine) [reply packages/slate-react/src/index.ts]: Idk, my editor has many errors. I've removed it, since it builds without it.

Screenshot 2019-04-02 at 11 51 20

**Comment by Soreine** [comment]
I have made it a package 👍

**Pull request merged**